data_steward/utils/pipeline_logging.py

```diff
@@ -111,3 +121,58 @@ def setup_logger(log_filepath_list, console_logging=True):
     for filename in log_path:
         log_list.append(create_logger(filename, console_logging))
     return log_list
+
+
+def get_default_log_path(logger_name):
+    """
+    Get the log file path associated with a logger
+    with the specified name
+
+    :param logger_name: name of the logger
+    :return: absolute path to the log file
+    """
+    return os.path.join(DEFAULT_LOG_DIR, f'{logger_name}.log')
+
+
+def get_logger(logger_name):
+    """
+    Get a logger with the specified name, creating it if necessary.
+    The logger writes >= INFO logs to stderr and >=DEBUG logs to 
+    a file at a standard location `DEFAULT_LOG_DIR`.
+
+    :param logger_name: name of the logger (usually set to __name__)
+    :return: the logger
+    :example:
+    >>> LOGGER = get_logger(__name__)
+    >>> # Additional handlers can be added if needed
+    >>> LOGGER.addHandler(logging.FileHandler('my_custom.log'))
+    >>> def func(p1, p2):
+    >>>     LOGGER.debug(f"func called with {p1}, {p2}")
+    """
+    logger = logging.getLogger(logger_name)
+    # explicitly set logger level to DEBUG otherwise
+    # the level would be that of the closest ancestor
+    logger.setLevel(logging.DEBUG)
+    # prevent inheriting handlers from ancestors
+    logger.propagate = False
```

Comment:
This logging setup makes for some difficult log reading.  With this line and lines 166 and 176 as written, the console looks like you would expect it to, all log messages being displayed as they are received.  The log files are hard to read.  I set up a test with the following in a file
```
import logging
import tools.exclude_site_submission as ess
from utils import pipeline_logging

# assuming a pipeline setup like yours was used in this fake module too
logger = pipeline_logging.get_logger(__name__)

logger.info("Random logging message")

if __name__ == '__main__':
    logger.info("STARTINGGGGGGGGG")
    # calling your code like referencing it from another module
    ess.main('/Users/lbiggers/.keys/aou-res-curation-test-me.json', 'aou-res-curation-test', 'lrwb_aou_ehr_ops_test', ['ca', 'fl'])
    logger.info("DONEEEEEE")
```
When I run this with the code, I get two separate log files, one titled `logs/__main__.py` and one titled `logs/tools.exclude_site_submission.log`.  Both files have log lines, but even though `logs/__main__.py` called the other module, neither module has logs from the other file.  So it doesn't look like the console with a "stream of conciousness" type logging.  To get that I would have to mix the two log files.

To get a single log file with the interwoven log statements, I had to remove this restriction and change the lines below from `logger.addHandler` to `logging.getLogger('').addHandler`.  Similar to this portion of the [logging cookbook](https://docs.python.org/3/howto/logging-cookbook.html#logging-to-multiple-destinations).

What issues were you addressing with this setup?